frontend/src/main/web/lib/components/glossary/SourceActionCell.jsx

```diff
@@ -56,11 +56,12 @@ var SourceActionCell = React.createClass({
     if(this.props.id === null || this.state.entry === null) {
       return <LoadingCell/>;
     } else {
+      var classes = this.props.info === Actions.NO_INFO_MESSAGE ? 'csec50' : 'cpri';
```

Comment:
I usually just call this `className` so that it consistently uses that name whenever it is talking about css classes - might make it a bit quicker for us to read these components in the long run.